### DISCUSSION TITLE
how to remove phishing link github?

### DISCUSSION THREAD

**Discussion by dewakalkun** [community/community#139214] [Repositories] [answered]
### Select Topic Area

Question

### Body

phising link: https://francodisasi.github.io/amp/

does anyone know how to report this phishing link. I have contacted support and no results
maybe github supports phishing / scam actions

**Comment by fikri-taufiqurrahman** [comment, marked as answer]
If you believe you've encountered a phishing or malicious link hosted on GitHub Pages (like the one you mentioned), here’s what you can do to report it properly:

Use GitHub's Abuse Report Form:

**Go to [GitHub's Abuse Report page](https://github.com/contact/report-abuse).**

Provide as much information as possible about the phishing link, including the URL, any suspicious activity, and your interactions (if any).
Select "GitHub Pages" as the category for the report, as the link is hosted on a GitHub Pages domain.
Email GitHub Support:

**You can also email GitHub at support@github.com or abuse@github.com directly with the phishing link and details.**


*It may take some time for GitHub to review your report. If they find that the content violates their terms of service, they will take it down.
*GitHub does not support phishing or scam activities, but investigations can take time, and more reports may speed up the process.
